### ISSUE TITLE
Unable to install on Android emulator

### ISSUE THREAD

**Issue by jacquesprogram** [UXDivers/Gorilla-Player-Support#305]
Hello,
I've installed Gorilla Player desktop app and Visual Studio 2017 addin as well. I'm trying now to install Gorilla on the android emulator provided by VS ( 'Android_Accelerated_Nougat (Android 7.1 API25)' ). The emulator launches and the deployment process starts but nothing happens on the emulator, gorilla player never starts.
The deployment process keeps going in a neverending loop... 
I don't know what I'm doing wrong...

**Comment by jadenrogers** [comment]
@jacquesprogram You could just have an issue with your emulator. Does the emulator actually run, as in you see the typical home screen for Android? Android emulators can be troublesome all by them self, deploy to a real device to check.

**Comment by profpjmg** [comment]
Hi,
I have the same problem.
I've tried several android emulators including a real device and none of them worked.
The connection to the server works and the gorilla icon appears in the xaml file.
Someone can help me?

**Comment by profpjmg** [comment]
Thanks for your answer!
I  have installed the UXDivers.GorillaPlayer.SDK.Autoconfig
I got the following errors:

![errors](https://user-images.githubusercontent.com/33670458/51872213-8bb8a300-2350-11e9-93c1-17b92b13d7b9.PNG)

Anything missing...

**Comment by jadenrogers** [comment]
@profpjmg Yes just for getting started remove the autoconfig package and just install

UXDivers.GorillaPlayer.SDK

    Install-Package UXDivers.GorillaPlayer.SDK -Version 1.5.0

**Comment by profpjmg** [comment]
I have got this:
![capturar](https://user-images.githubusercontent.com/33670458/51873665-00421080-2356-11e9-82de-2340099035bf.PNG)

He ask me for the server ip. Where can I find it?

**Comment by jadenrogers** [comment]
@profpjmg 

Good stuff. First make sure Gorilla player server is running on your dev machine.
Make sure your device is on the same local network.

If it doesn't detect the server automatically get your PC local IP by right clicking the Gorilla Player tray icon and selecting "About Gorilla Player"

Then "Show Details" and you should see your IP address. Punch it into the app and hit connect!

![image](https://user-images.githubusercontent.com/8000621/51874083-66fe1300-23c4-11e9-8d8b-764d3b4d575d.png)


![image](https://user-images.githubusercontent.com/8000621/51874086-6bc2c700-23c4-11e9-84fb-032f6745a182.png)

**Comment by profpjmg** [comment]
I don't find my local IP  in the Gorilla Player because the aplication with right clicking in the icon don't show me nothing. I find him in the internet. After entering the IP (starts 192.168....) I got the following image:

![error2](https://user-images.githubusercontent.com/33670458/51875501-28813d80-235d-11e9-9a51-ce4435b4e78d.PNG)

Thank you for the patience.

**Comment by profpjmg** [comment]
I find my local IP like you have said and I have disconnect the firewall of W10 and Kaspersky, he still give me the same error: time out.
I try a seconde time with _my local IP:9014_ I think it works but the previewr does not make updates.

**Comment by profpjmg** [comment]
It works!
I forgot to save changes,
Thank you very much

**Comment by albertoms** [comment]
I had the same problem with the Android emulator on a Mac. The screen was just looping waiting for a server to connect with.

I typed `ifconfig` on my terminal and looked up for the `e0` block:
```
en0: flags=8863<UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST> mtu 1500
	ether 8c:85:90:40:1f:5a
	inet6 fe80::4a6:dba7:5f25:bf5c%en0 prefixlen 64 secured scopeid 0x6
	inet 192.168.100.149 netmask 0xffffff00 broadcast 192.168.100.255
	nd6 options=201<PERFORMNUD,DAD>
	media: autoselect
	status: active
```

Then I copied the IP from the `inet` line and pasted it on the Android emulator. It works now :)
